packages/components/src/date-time/types.ts

```diff
@@ -0,0 +1,72 @@
+export type DateTimeString = string;
+
+export type ValidDateTimeInput = Date | string | number | null;
```

Comment:
There's a chance that we'll have to "expand" this type intro each prop type,. since the docgen used by Storybook is usually unable to expand these types when generating docs.

We can also decided to tackle this in a later PR, when working on this component's Storybook examples.

Reply:
Bugger! I love using type aliases as a form of documentation.

Reply:
Same! It's a limitation of the docgen and we're hoping that it's going to be improved soon